src/spikeinterface/extractors/neoextractors/spikegadgets.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, file_path, stream_id=None, stream_name=None, block_index=None
             probegroup = probeinterface.read_spikegadgets(file_path, raise_error=False)
 
         if probegroup is not None:
-            self.set_probes(probegroup, in_place=True)
+            self._set_probes(probegroup, in_place=True)
```

Comment:
We should change this to use set_probe_group, right?

Reply:
Not necessarily. This is a private call to the function so we are entitled to use the private or public api, no?

Reply:
I am just thinking that the code will be more specific.

Reply:
I guess I don't know gadgets well enough. We call it a probegroup, but I don't know if it could ever return a probe instead. So set_probes protects us from that case that the variable was named poorly. I guess I should look up the actual read_spikegadgets and see if it is forced to truly be a probegroup.